### DISCUSSION TITLE
How to change the default database password

### DISCUSSION THREAD

**Discussion by dabuv** [pglombardo/PasswordPusher#431] [Q&A] [answered]
Hi,
I would like to change default DB password specified in docker compose file but every time i deploy it, the app container can't start properly because it cannot connect to DB. Is there any solution for this? It's probably a novice question but answer would also be helpful for others

**Comment by pglombardo** [comment]
If you change the password, you also have to change the `DATABASE_URL` environment variable in the app container.  If that doesn't work, post the backtrace and I'll take a look.

**Comment by pglombardo** [comment, marked as answer]
This is the line to change:
https://github.com/pglombardo/PasswordPusher/blob/master/containers/docker/pwpush-postgres/Dockerfile#L8

or alternatively, override the environment variable entirely when launching the container (in the docker-compose.yml file).

**Reply by dabuv** [reply]
Ohhh... I will give it a try and provide a feedback. Thanks for Your time and this amazing tool 👍
